.github/workflows/merge-staging.yml

```diff
@@ -0,0 +1,30 @@
+name: "merge staging(-next)"
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # Merge every 6 hours
+    - cron:  '* */6 * * *'
```

Comment:
How about publishing merge conflicts notifications on IRC?

Reply:
That's a good idea. Notifying on IRC could be done with e.g. https://github.com/marketplace/actions/irc-message-action